inc/ThirdParty/Hostings/HostSubscriberFactory.php

```diff
@@ -45,6 +47,9 @@ public function get_subscriber() {
 				return new Godaddy();
 			case 'kinsta':
 				return new Kinsta();
+			case 'onecom':
+				$options = new Options( 'wp_rocket_' );
+				return new OneCom( $options,  new Options_Data( $options->get( 'settings', [] ) ) );
```

Comment:
I believe it'd be cleaner to use the helper functions `get_rocket_option` and `update_rocket_option` declared here:
https://github.com/wp-media/wp-rocket/blob/075a6013f6377c4cad2f56a11f10211e7541b303/inc/functions/options.php#L8-L41

instead of passing the options instance.